contents/blog/introducing-hogql.md

```diff
@@ -0,0 +1,98 @@
+
+SQL insights are especially useful for when you need to do exotic or novel analysis that isn't immediately possible with existing insight types. For example, while you can use a retention insight to discover which product features keep users coming back for more, you could use an SQL insight to find which users keep coming back and even identify outliers. 
+
+We're incredibly excited to hear your feedback and see what's possible with SQL access in PostHog, and believe it will give engineers a unique edge and better information than they'd be able to get with other product analytics tools.
+
+## Custom insights
+The final new insight type is called Custom insights. Like the SQL insight type, custom insights enable you to create a very wide range of insights and visualizations - but it differs because it's based on JSON, rather than SQL. 
+
+## The PostHog event explorer 
+In addition to introducing HogQL, we've also upgraded the event explorer (previously known as the Live events tab) to give you more ways to filter events and analyze your data.
+
+Unlike the old "Live events" view, the event explorer lets you use HogQL expressions (`properties.$screen_width * $properties.screen_height`) and aggregations (`sum(properties.price)`) as columns.
+
+The event explorer also lets you view and edit the JSON source behind the table. We've also added the ability to add custom columns and filter them using HogQL. This is helpful not just for generating tables which summarize person and event information according to your needs, but also for exploring the data in new and totally custom ways. You could, for example, use HogQL to run currency conversions and normalize data into a single currency in its own column - or even extrapolate and project revenue and usage figures into the future.
+
+Below you can see a query we wrote using HogQL which summarizes [votes from our 404 page](/hedgehogsgalore).
+
+```
+{
+  "kind": "DataTableNode",
+  "full": true,
+  "source": {
+    "kind": "EventsQuery",
+    "select": [
+      "properties.$geoip_country_name",
+      "if(countIf(properties.does_pineapple_go_on_pizza == 'false') > countIf(properties.does_pineapple_go_on_pizza == 'true'), '🍅 Does not belong on pizza', if(countIf(properties.does_pineapple_go_on_pizza == 'false') == countIf(properties.does_pineapple_go_on_pizza == 'true'), '🥦 It is a tie' , '🍍 Belongs on pizza')) -- Result",
+      "concat(repeat('🍍', countIf(properties.does_pineapple_go_on_pizza == 'true')),repeat('🍅', countIf(properties.does_pineapple_go_on_pizza == 'false'))) -- Answers",
+      "count() -- Number of answers",
+      "concat(toString(round(countIf(properties.does_pineapple_go_on_pizza == 'true') / count() * 1000) / 10), '%') -- Percentage pineapple"
+    ],
+    "after": "-365d",
+    "limit": 100,
+    "event": "pineapple_on_pizza_survey",
+    "orderBy": [
+      "-count() # Number of answers"
+    ],
+    "before": ""
+  },
+  "propertiesViaUrl": true
+}
+```
```

Comment:
I think this example would be [better as SQL](https://app.posthog.com/insights/K84UWNwR).

Reply:
"Insight not found" ?